routes/index.js

```diff
@@ -1,8 +1,141 @@
 var express = require('express');
+var passport = require('passport');
 var router = express.Router();
+var User = require('../models/index.js').User;
 
 router.get('/', function(req, res, next) {
-  res.render('index', { title: 'Express' });
+  var user = req.user || null;
```

Comment:
This part feels a little strange to me. Why are we passing along a user on the request object and sending that user back?